databroker/databroker.py

```diff
@@ -423,3 +423,45 @@ def get_table(headers, fields=None, fill=True, convert_times=True):
     else:
         # edge case: no data
         return pd.DataFrame()
+
+
+def stream(headers, fields=None, fill=True):
+    """
+    Get all Documents from given run(s).
+
+
+    Parameters
+    ----------
+    headers : Header or iterable of Headers
+        The headers to fetch the events for
+    fields : list, optional
+        whitelist of field names of interest; if None, all are returned
+    fill : bool, optional
+        Whether externally-stored data should be filled in. Defaults to True
+
+    Yields
+    ------
+    doc : Document
```

Comment:
I think this should `yield name, doc` as that is the signature that the call backs expect.